client/src/components/DeleteModal.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Button, Modal, Header } from 'semantic-ui-react';
+
+import { deleteSession } from '../utils/apiWrapper.js';
+
+import 'semantic-ui-css/semantic.min.css';
+import '../css/DeleteModal.scss';
+
+function DeleteModal({ isActive, creator, id }) {
+  const [open, setOpen] = React.useState(false);
+
+  const handleEndAndDelete = async () => {
+    const response = await deleteSession(id);
+    console.log(response);
+  };
+
+  return (
+    <Modal
+      onClose={() => setOpen(false)}
+      onOpen={() => setOpen(true)}
+      open={open}
+      trigger={
+        <Button
+          className={'join-leave-btn'}
+          size="small"
+          onClick={handleEndAndDelete}
+          content={isActive ? 'END' : 'CANCEL'}
+        />
```

Comment:
Right now, if you click the End/Cancel button, the Details Modal also pops up underneath. This is because I changed the Details Modal to pop up when a user clicks a Session Summary card. To fix this, you have to add a Stop Propagation to the onClick function of any buttons on the SessionSummary card so that the Details Modal does not pop up when you click the button.

Change this button's onClick to this below:

```
          onClick={(e) => {
            e.stopPropagation();
          }}
```

You have to remove handleEndAndDelete and move it to the button on lines 37-39 because you don't want to delete/end a session hen a user presses the "End" button on the Session Summary. You want to end the session when the user presses the button in the modal. This button above (line 23-28) is just a trigger to open the modal, it's not the true delete button that users click after seeing "Are you sure you want to delete the session?"